File: Iniciante/2542/2542.py
while True:
    try:
        n = int(input())
        m, l = map(int, input().split())
        matrizM = []
        matrizL = []
        for i in range(m):
            linha = [int(k) for k in input().split()]
            matrizM.append(linha)
        for j in range(l):
            linha = [int(k) for k in input().split()]
            matrizL.append(linha)
        cm , cl = map(int, input().split())
        cm -= 1
        cl -= 1
        atributo = int(input()) -1
        if matrizM[cm][atributo] > matrizL[cl][atributo]:
            print("Marcos")
        elif matrizM[cm][atributo] == matrizL[cl][atributo]:
            print("Empate")
        else:
            print("Leonardo")
    except EOFError:
        break

File: Iniciante/2542/2542.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int n, m, l, cm, cl, atributo;
    while (cin >> n){
        cin >> m >> l;
        int marcos[m][n], leo[l][n];
        for(int i = 0; i<m; i++){
            for(int j = 0; j<n; j++) cin >> marcos[i][j];
        }
        for(int i = 0; i<l; i++){
            for(int j = 0; j<n; j++) cin >> leo[i][j];
        }
        cin >> cm >> cl;
        cm--;
        cl--;
        cin >> atributo;
        atributo--;
        if (marcos[cm][atributo]>leo[cl][atributo]) cout << "Marcos" << endl;
        else if (marcos[cm][atributo] == leo[cl][atributo]) cout << "Empate" << endl;
        else cout << "Leonardo" << endl;
    }
    return 0;
}